src/hooks/useStorybookLog.ts

```diff
@@ -0,0 +1,19 @@
+import { reactive } from '@muban/muban';
+
+/**
+ * This hook can be used to keep track of logs within Storybook
+ *
+ * @returns
+ */
+export function useStorybookLog(): { state: Array<string>; log: (message: string) => void } {
+  const state = reactive<Array<string>>([]);
+
+  function log(message: string): void {
+    state.push(message);
+    setTimeout(() => {
+      state.splice(0, 1);
+    }, 2000);
+  }
+
+  return { state, log };
```

Comment:
Maybe rename to `logState` (or `logs`) to be a bit less generic when using this in other code?

Reply:
Wouldn't you just destructure with a unique name if it's conflicting?

Reply:
> Wouldn't you just destructure with a unique name if it's conflicting?

Sure, something you could always do, but often only used for "edge cases", ideally you want to have the default name be usable in default cases.

It's not just for conflicts, but also because it's generic/confusing :D

When exposing an array of options (like `[state, setState]`), you would always give them a name when used, so there it's less relevant. Here, since it's an object, you want usable and sensible default names :)